pom.xml

```diff
@@ -0,0 +1,678 @@
+                            <arguments>
+                                <argument>--prefix=/</argument>
+                                <argument>--libdir=/</argument>
+                            </arguments>
+                            <workingDirectory>${project.build.directory}/${project.artifactId}/src/main/c</workingDirectory>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>make</id>
+                        <phase>process-sources</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>make</executable>
+                            <arguments>
+                                <argument>-j1</argument>
+                            </arguments>
+                            <workingDirectory>${project.build.directory}/${project.artifactId}/src/main/c</workingDirectory>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>make-install</id>
+                        <phase>process-sources</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>make</executable>
+                            <arguments>
+                                <argument>-j1</argument>
+                                <argument>install</argument>
+                                <argument>DESTDIR=${project.build.directory}/classes</argument>
+                            </arguments>
+                            <workingDirectory>${project.build.directory}/${project.artifactId}/src/main/c</workingDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.3.0</version>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <addClasspath>true</addClasspath>
+                            <!-- <classpathPrefix>lib</classpathPrefix> -->
+                            <!-- <mainClass>test.org.Cliente</mainClass> -->
+                        </manifest>
+                        <manifestEntries>
+                            <Class-Path>lib/javalognorm/</Class-Path>
+                        </manifestEntries>
+                    </archive>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
+                <version>2.2.1</version>
```

Comment:
I would like to use newer version such as 3.3.0

Reply:
Solved in commit 79e7942bab31d31302d436da42c0e065ce22da57